game/main.lua

```diff
@@ -1,4 +1,7 @@
 
+-- LUX portability
+require 'lux.portable'
```

Comment:
I was gonna suggest we did the things this file does eventually. Glad it's done now